### ISSUE TITLE
How to design this graph?

### ISSUE THREAD

**Issue by MarkJackson1997** [FormidableLabs/victory-native-xl#212]
### Question

I want to design barchart with different color,add unfill color and need to hide y-axis label.Please check attached image for better understanding.Please help

### Background Info/Attempts
I checked this example but how to set dynamic color in this?and also want to hide y-axis label
<img width="167" alt="Screenshot 2024-03-14 at 12 34 24 AM" src="https://github.com/FormidableLabs/victory-native-xl/assets/39664330/b471199f-d596-49ae-9b1f-8332d87f4a6e">

 <CartesianChart
            xKey="day"
            padding={10}
            yKeys={["stars"]}
            domainPadding={20}
            axisOptions={{
              font,
              tickCount: 5,
              lineColor: isDark ? "#71717a" : "#d4d4d8",
              labelColor: isDark ? appColors.text.dark : appColors.text.light,
            }}
            data={DATA}
          >
            {({ points }) => {
              return (
                <>
                  {points.stars.map(({ x, y }) => {
                    return (
                      <React.Fragment key={`point-${x}-${y}`}>
                        <Points
                          points={calculateStarPoints(x, y ?? 0, 5, numPoints)}
                          mode="polygon"
                          color={shapeColor}
                          strokeCap="round"
                          strokeWidth={2}
                        />
                      </React.Fragment>
                    );
                  })}
                </>
              );
            }}
          </CartesianChart>

**Comment by filiphammarnberg** [comment]
Change `tickCount: 5` to `tickCount: { x: 5, y: 0 }` to get rid of the ticks on the y axis.

I'm not quite sure why you are using a Points component? Please see this guide for a similar chart as your provided screenshot: https://commerce.nearform.com/open-source/victory-native/cartesian/guides/basic-bar-chart

**Comment by MarkJackson1997** [comment]
@ffoton yes if i using y:0 then it will hide y axis,i am trying to set different colors and unfill color on bar for this i am checing example code.In that used Points component.
For better understanding please check attached image in question.
In that 2 different color on bar chart and also set unfill color gray.
